### PR TITLE
Implementation of activation functions

### LINKED ISSUE #2: Implement the ReLU function
Make a simple python implementation for the ReLU activation function to be used in a multilayer neural network.

### LINKED ISSUE #3: Implement hyperbolic tangent (tanh) function
Make a simple implementation of tanh activation function

### LINKED ISSUE #4: implement sigmoid function
Implement the sigmoid activation function

### LINKED ISSUE #5: Implement softmax function
implement a simple softmax to be used in multilayer networks

### LINKED ISSUE #6: Implement leaky ReLU

### LINKED ISSUE #13: Backpropagation
Backpropagation algorithm is used to train a multilayer neural network, updating the weights inside the network, it aims to reduce the error in the generalization and cause the net to converge.

Backpropagation

### PR DESCRIPTION
**Pull request by JT4v4res**

### CODE REVIEW

**Comment by JT4v4res** [comment]
Implemented the activation functions with the derivative parameter for best use in backpropagation.
